_data/sites.json

```
@@ -2383,6 +2383,16 @@
         ]
     },
 
+    {
+        "name": "Coin",
+        "url": "https://coinapp.co",
+        "difficulty": "hard",
+        "notes": "Have to contact them, use <a href=\"https://support.coinapp.co/hc/en-us/requests/new\">https://support.coinapp.co/hc/en-us/requests/new</a> and ask for account to be deleted",
```

```suggestion
        "url": "https://support.coinapp.co/hc/en-us/requests/new",
        "difficulty": "hard",
        "notes": "Contact them via the ticketing system and ask for account to be deleted",
```
We try to put the URL to be as direct as possible to the deletion location